tests/run/xml-interpolation/XmlQuote_1.scala

```diff
@@ -0,0 +1,52 @@
+import scala.quoted._
+import scala.tasty.Tasty
+
+case class Xml(parts: String, arg: Any)
+
+class XmlQuote(ctx: StringContext) {
```

Comment:
Shouldn't this class extend `AnyVal`?

Reply:
I doesn't matter for performance. The code is elided by the macro. So the simpler, the better

Reply:
Even the allocation of the XmlQuote is elided?

Reply:
Actually my intuition (based on Scala 2 macros work) was wrong. The receiver of the macro doesn't get elided. I opened #5110